doc/md/monitoring/02-quickstart.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Quickstart
+---
+A quick and easy to follow guide to get started.
+
+Prerequisites:
```

Comment:
```suggestion
### Prerequisites:
```

font should be header
![image](https://github.com/user-attachments/assets/38848f15-81c8-4c1e-bb60-789cb2c6c563)

Reply:
I think in all other places it is simple text as well. So I prefer to keep it consistent.